app/src/main/java/com/teamwss/websoso/ui/main/record/RecordAdapter.kt

```diff
@@ -0,0 +1,28 @@
+package com.teamwss.websoso.ui.main.record
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.teamwss.websoso.data.Memo
+import com.teamwss.websoso.databinding.ItemMemoBinding
+
+class RecordAdapter(context: Context) : RecyclerView.Adapter<RecordViewHolder>() {
+    private val inflater by lazy { LayoutInflater.from(context) }
+    private var memoList: List<Memo> = emptyList()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecordViewHolder {
+        val binding = ItemMemoBinding.inflate(inflater, parent, false)
+        return RecordViewHolder(binding)
+    }
```

Comment:
```suggestion
override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecordViewHolder {
    val inflater = LayoutInflater.from(parent.context)
    val binding = ItemMemoBinding.inflate(inflater, parent, false)
    return RecordViewHolder(binding)
}
```
Adapter의 매개변수로 context를 받지 않고 `LayoutInflater.from(parent.context)`로 받아오는 것이 좋을 것 같습니다.

그리고 `RecordViewHolder` 인스턴스를 `RecordAdapter`에서 만들고 있는데 `RecordViewHolder`에서 만드는 방법을 알아보시는 것이 좋을 것 같습니다. 

제 3차과제 PR과 공부했어요 노션을 참고해주세요

Reply:
제가 정확하게 이해한건지 모르겠지만 viewholder와 adapter를 분리하여 viewholder 내부에서 viewholder를 직접 선언하고 가독성과 유지보수성을 향상시키는 방법이 맞을까요? 코드 수정했습니다! 확인해보시고 답변 부탁드립니다!

Reply:
데일리 스크럼 끝나고 설명드리겠습니다!